sql/core/src/test/resources/sql-tests/inputs/pgSQL/with.sql

```diff
@@ -0,0 +1,1215 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- WITH
+-- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/with.sql
+--
+-- This test suite contains two Cartesian products without using explicit CROSS JOIN syntax.
+-- Thus, we set spark.sql.crossJoin.enabled to true.
+set spark.sql.crossJoin.enabled=true;
```

Comment:
This related to my comment there: https://github.com/apache/spark/pull/25092#discussion_r302319639 cc: @wangyum

Reply:
@dongjoon-hyun @maropu I filed a JIRA:
https://issues.apache.org/jira/browse/SPARK-28343

Reply:
Great. Thanks. 
@maropu . Please update that JIRA if we need more.